src/TouchCarousel.js

```diff
@@ -55,22 +55,37 @@ class TouchCarousel extends React.PureComponent {
     this.grabbing = false
     this.tracingTouchId = null
     this.isMovingCross = null
+    this.isMouseDown = false
+    this.mouseTouchId = 0
   }
 
   componentDidMount () {
+    document.addEventListener('mousemove', this.onDocumentMouseMove)
+    document.addEventListener('mouseup', this.onDocumentMouseUp)
```

Comment:
It would be nice if we add a prop and make all the mouse support stuff opt-in. It's "touch-carousel" after all.